### PR TITLE
Account for project-onboarding renaming

### PR DESCRIPTION
**Pull request by tobie**
This follows #641.

### CODE REVIEW

File: PROJECT_PROGRESSION.md
Comment (ljharb) [inline PROJECT_PROGRESSION.md:90]:
```suggestion
<!--If this checklist is updated please ensure `https://github.com/openjs-foundation/project-status/blob/HEAD/ISSUE_TEMPLATE/00-project-onboarding-checklist-template.md` is updated as well -->
might as well avoid a future link edit too :-)
```

**Review by ljharb — APPROVED** [review]

File: PROJECT_PROGRESSION.md
Comment (tobie) [inline PROJECT_PROGRESSION.md:90]:
```suggestion
<!--If this checklist is updated please ensure `https://github.com/openjs-foundation/project-status/blob/HEAD/.github/ISSUE_TEMPLATE/00-project-onboarding-checklist-template.md` is updated as well -->
```

**Comment by tobie** [comment]
Blocked on https://github.com/openjs-foundation/project-status/pull/53.

**Review by kborchers — APPROVED** [review]

**Comment by MylesBorins** [comment]
@tobie should this maybe just be rolled into https://github.com/openjs-foundation/project-status/pull/53 to avoid confusion?

**Review by MylesBorins — APPROVED** [review]
LGTM

**Comment by tobie** [comment]
> @tobie should this maybe just be rolled into [openjs-foundation/project-status#53](https://github.com/openjs-foundation/project-status/pull/53) to avoid confusion?

These are different repos, unfortunately.

**Review by dmethvin — APPROVED** [review]

**Comment by MylesBorins** [comment]
@tobie LOL oops 😂

Thanks for makign me aware of the obvious

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by sendilkumarn — APPROVED** [review]

**Review by mmarchini — APPROVED** [review]

**Review by joesepi — APPROVED** [review]

**Review by eemeli — APPROVED** [review]

**Comment by mhdawson** [comment]
This has been open for more than the required time, has enough approvals, landing.

**Pull request merged**
